streams/src/main/java/org/apache/kafka/streams/Topology.java

```diff
@@ -510,62 +517,38 @@ public synchronized <K, V> Topology addSink(final String name,
     }
 
     /**
-     * Add a sink that sends records from upstream
-     * {@link #addProcessor(String, ProcessorSupplier, String...) processors} or
-     * {@link #addSource(String, String...) sources} to Kafka topics based on the provided {@link TopicNameExtractor}.
-     * All topics that the topic name extractor may compute should be created before the {@link KafkaStreams} instance
-     * is started.
-     *
-     * <p>The sink will use the default values from {@link StreamsConfig} for
-     * <ul>
-     *   <li>{@link StreamsConfig#DEFAULT_KEY_SERDE_CLASS_CONFIG key serializer}</li>
-     *   <li>{@link StreamsConfig#DEFAULT_VALUE_SERDE_CLASS_CONFIG value serializer}</li>
-     * </ul>
-     *
-     * Furthermore, the producer's configured partitioner is used to write into the topic.
-     * If you want to specify a sink specific key or value {@link Serializer}, or use a different
-     * {@link StreamPartitioner partitioner}, use the corresponding overloaded {@code addSink(...)} method.
-     *
-     * @param name
-     *        the unique name of the sink
-     * @param topicExtractor
-     *        a topic name extractor to determine the name of the Kafka topic to which this sink should write for each record
-     * @param parentNames
-     *        the name of one or more {@link #addProcessor(String, ProcessorSupplier, String...) processors} or
-     *        {@link #addSource(String, String...) sources}, whose output records this sink should consume and write
-     *        to the specified output topic
-     *
-     * @return itself
-     *
-     * @throws TopologyException
-     *         if the provided sink name is not unique, or
-     *         if a parent processor/source name is unknown
+     * See {@link #addSink(String, String, String...)}.
```

Comment:
Realized that this JavaDocs is 99% the same, so remove it here and just add a link.